src/Makefile

```diff
@@ -496,8 +498,8 @@ config-sanity:
 	@test "$(pext)" = "yes" || test "$(pext)" = "no"
 	@test "$(comp)" = "gcc" || test "$(comp)" = "icc" || test "$(comp)" = "mingw" || test "$(comp)" = "clang"
 
-$(EXE): $(OBJS)
-	$(CXX) -o $@ $(OBJS) $(LDFLAGS)
+$(EXE): $(notdir $(SRCS:.cpp=.o))
+	$(CXX) -o $@ $(notdir $(SRCS:.cpp=.o)) $(LDFLAGS)
```

Comment:
Because we defined OBJS above we can omit these changes.
```Makefile
$(EXE): $(OBJS)
	$(CXX) -o $@ $(OBJS) $(LDFLAGS)
```

Reply:
Yes, defining OBJS seems like a good idea, makes the exe rule clearer. I'll do that ...